### PR TITLE
Explicit units and meaning of returned DataArrays

### PR DESCRIPTION
**Pull request by euronion**
I'm not 100% satisfied with the PR, it feels a bit hacky. Nevertheless...

## Change proposed in this Pull Request

Explicitly provide units and a name for returned DataArrays created by `.wind(...)` and `.pv(...)` function.

## Description

For `.wind(...)` the location to set name and unit is straightforward.
For `.pv(...)` I'm not an expert for the Boilfinger model and couldn't find the correct location; I would assume the unit is identical to the Huld model, but since I am not 100% sure I decide to set unit and name inside the specific model.

Inside `.convert_and_aggregate(...)` where the DataArrays are manipulated the unit/name are either changed or keeping of the attributes is made explicit via `keep_attrs=True` (otherwise they would be implicitly removed by `xarray`).

Caveat: Setting of units relies on some conventions, e.g. for the units assumed in the panelconfig and turbineconfig files. However the code already makes these assumptions, so it is not a new hard-coded reliance that is introduced.

## Motivation and Context

Units until now were not made explicit for these cases and confusing.

## How Has This Been Tested?
Local testing.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] I tested my contribution locally and it seems to work fine.
- [ ] I locally ran `pytest` inside the repository and no unexpected problems came up.
- [ ] I have adjusted the docstrings in the code appropriately.
- [ ] I have documented the effects of my code changes in the documentation `doc/`.
- [ ] I have added newly introduced dependencies to `environment.yaml` file.
- [x] I have added a note to release notes `doc/release_notes.rst`.
- [x] I have used `pre-commit run --all` to lint/format/check my contribution

### CODE REVIEW

File: atlite/convert.py [hunk 1]
```diff
@@ -120,9 +120,15 @@ def convert_and_aggregate(
                 "given for `per_unit` or `return_capacity`"
             )
         if capacity_factor:
-            return maybe_progressbar(da.mean("time"), show_progress, **dask_kwargs)
+            return maybe_progressbar(
+                da.mean("time").assign_attrs(units="p.u.").rename("capacity factor"),
+                show_progress,
+                **dask_kwargs,
+            )
```
Comment (FabianHofmann) [inline atlite/convert.py:127]:
```suggestion
            res = da.mean("time").rename("capacity factor")
            res.attrs["units"] = "p.u."
            return maybe_progressbar(res, show_progress, **dask_kwargs)
```

File: atlite/convert.py [hunk 2]
```diff
@@ -120,9 +120,15 @@ def convert_and_aggregate(
...
         else:
-            return maybe_progressbar(da.sum("time"), show_progress, **dask_kwargs)
+            return maybe_progressbar(
+                da.sum("time", keep_attrs=True), show_progress, **dask_kwargs
+            )
```
Comment (FabianHofmann) [inline atlite/convert.py:131]:
```suggestion
            res = da.sum("time", keep_attrs=True)
            return maybe_progressbar(res,  show_progress, **dask_kwargs)
```

**Review by FabianHofmann — COMMENTED** [review]
Great! Good that you have a look at that! Just a picky request on aligning the code style in the convert function to the other parts with returns

**Comment by euronion** [comment]
Picky is welcome ;-)

**Pull request merged**